### DISCUSSION TITLE
Lorem Ipsum Generator

### DISCUSSION THREAD

**Discussion by WeirDave** [phcode-dev/phoenix#1746] [Q&A] [answered]
It would be helpful to have a Lorem Ipsum generator to generate paragraphs of text for use in creating websites:

Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. In dictum non consectetur a erat nam at lectus. Erat pellentesque adipiscing commodo elit. Nibh venenatis cras sed felis eget velit. Nec sagittis aliquam malesuada bibendum. Pulvinar elementum integer enim neque volutpat ac tincidunt vitae. Id venenatis a condimentum vitae sapien pellentesque habitant morbi tristique. Risus ultricies tristique nulla aliquet enim. Diam vel quam elementum pulvinar etiam non quam lacus suspendisse. Ultrices in iaculis nunc sed augue lacus viverra. Sed viverra tellus in hac habitasse platea dictumst. Massa placerat duis ultricies lacus sed turpis tincidunt id. Aliquet enim tortor at auctor urna nunc id cursus. Ac odio tempor orci dapibus ultrices in iaculis. Elit pellentesque habitant morbi tristique senectus et. Arcu non sodales neque sodales ut etiam sit. Integer enim neque volutpat ac.

Nec sagittis aliquam malesuada bibendum arcu. Facilisis sed odio morbi quis commodo odio aenean sed adipiscing. Quis imperdiet massa tincidunt nunc pulvinar sapien. Ultricies mi quis hendrerit dolor magna. Non quam lacus suspendisse faucibus. Vitae turpis massa sed elementum tempus egestas. Consectetur adipiscing elit pellentesque habitant morbi tristique. Augue ut lectus arcu bibendum at varius. Arcu dui vivamus arcu felis bibendum. Metus aliquam eleifend mi in nulla posuere sollicitudin aliquam ultrices. Augue eget arcu dictum varius duis at consectetur lorem donec.

**Comment by abose** [comment, marked as answer]
Goto `file - extension manager` . search for lorem. There should be a couple of extensions for that. Does that help @WeirDave ?

**Reply by WeirDave** [reply]
Cool! Thanks
